frontend/src/components/object/ObjectTable.vue

```diff
@@ -42,36 +47,31 @@ const emit = defineEmits(['show-object-info']);
 // Store
 const objectStore = useObjectStore();
 const permissionStore = usePermissionStore();
-const { getObjects } = storeToRefs(objectStore);
 const { getUserId } = storeToRefs(useAuthStore());
 
 // State
-const selectAll = ref(false);
 const permissionsVisible = ref(false);
 const permissionsObjectId = ref('');
 const permissionsObjectName: Ref<string | undefined> = ref('');
 const tableData: Ref<Array<COMSObjectDataSource>> = ref([]);
+const dt = ref();
```

Comment:
This variable name is not very self-evident. Would recommend a more verbose name change so that it becomes more clear at a glance what the purpose of this ref is supposed to be.